az-core/src/main/java/azkaban/Constants.java

```diff
@@ -92,6 +92,7 @@
 
     // Configures Azkaban to use new polling model for dispatching
     public static final String AZKABAN_POLL_MODEL = "azkaban.poll.model";
+    public static final String AZKABAN_POLLING_INTERVAL_MS = "azkaban.polling.interval.ms";
```

Comment:
is millisecond over fine-grained?

Reply:
The polling frequency will be adjusted based on how many flows are scheduled/executed at the same time. For example, if 100 flows are scheduled to run at the same time and we only have one executor, we might want to configure small polling intervals like 100 ms to reduce execution delays. In the future, it's very likely that we need to support more flows scheduled at the same time, so I think millisecond is a proper time unit for the polling interval.